astroplan/plots/finder.py

```diff
@@ -66,6 +68,12 @@ def plot_finder_image(target, survey='DSS', fov_radius=10*u.arcmin,
 
     hdu : `~astropy.io.fits.PrimaryHDU`
         FITS HDU of the retrieved image
+
+
+    Notes
+    -----
+    Dependencies:
+        In addition to Matplotlib, this function makes use of astroquery.
```

Comment:
I think this also depends on wcsaxes, right?
If yes, you should mention this dependency here also.